packages/configs/default/index.json

```diff
@@ -27,6 +27,7 @@
     "*.pug": ["@parcel/transformer-pug"],
     "*.coffee": ["@parcel/transformer-coffeescript"],
     "*.mdx": ["@parcel/transformer-mdx"],
+    "*.{png,jpg,jpeg,webp}": ["@parcel/transformer-image", "..."],
```

Comment:
So, currently if you want to import an image from JS you need to use the url: pipeline. This would make it so you do not. And if you did, then you wouldn’t get resizing capabilities because it wouldn’t use this pipeline... Should url: be required? For non JS it isn’t though, so not sure how we’d implement that.

Reply:
Not sure, we could add it if you prefer. Probably makes more sense to stay consistent